lib/queuePopulator/IngestionProducer.js

```diff
@@ -171,6 +186,9 @@ class IngestionProducer {
                         logRes,
                         initState: { status, versionMarker, keyMarker },
                     };
+                    if (initialCseq) {
```

Comment:
Is there a chance the `cseq` can be zero?

Reply:
not that I'm aware of. I've observed cseq to actually be greater than 1 even when there are no objects in a given bucket. The first few log entries are the bucket entries

Reply:
I recently learnt that cseq/aseq start from 1.